lib/sms.rb

```diff
@@ -0,0 +1,27 @@
+require 'twilio-ruby'
+
+class Sms
+
+  def initialize(client_klass = Twilio::REST::Client)
+    @client_klass = client_klass
+    @client = get_client
+  end
+
+  def send(msg)
+    from = ENV['TWILIO_NUMBER']
+    to = ENV['MY_NUMBER']
+    @client.account.messages.create(
+          :from => from,
+          :to =>  to,
+          :body => msg)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)